addons/sourcemod/scripting/shavit-core.sp

```diff
@@ -2068,6 +2113,19 @@ void StartTimer(int client, int track)
 	}
 }
 
+void SetStart(int client)
+{
+	if(!IsValidClient(client) || IsFakeClient(client))
+	{
+		return;
+	}
+	
+	Call_StartForward(gH_Forwards_SetStart);
```

Comment:
Why did you implement this as a forward? It'd make more sense to implement a native in shavit-zones that would handle that.